eng/containers/UbuntuNetCore2Keyring/ci.yml

```diff
@@ -0,0 +1,26 @@
+trigger:
```

Comment:
We should add a path filter for the trigger.

Reply:
Ok, how it should look like?

Reply:
You can use https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/ci.yml#L3 as a reference but I think all you need is:

```
trigger:
  branches:
    include:
    - master
  paths:
    include:
    - eng/containers/
```